rclrs/src/node.rs

```diff
@@ -71,10 +71,15 @@ pub struct Node {
 /// This struct manages the lifetime of an `rcl_node_t`, and accounts for its
 /// dependency on the lifetime of its `rcl_context_t` by ensuring that this
 /// dependency is [dropped after][1] the `rcl_node_t`.
+/// Note: we capture the rcl_node_t returned from rcl_get_zero_initialized_node()
+/// to guarantee that the lifetime of node handle exists until we drop the NodeHandle
```

Comment:
Something feels a bit off with the phrasing here, maybe something like the following:
```suggestion
/// to guarantee that the node handle exists until we drop the NodeHandle
```